### PR TITLE
Made error more specific, closes #542

### LINKED ISSUE #542: Twitter::Error in twitter:update_lists
## Exception in Hacken.in

**Twitter::Error** in **twitter:update_lists**
execution expired

[View on Bugsnag](https://bugsnag.com/hacken-dot-in/hacken-dot-in/errors/54b9e3380eb4aaa7d97168db?event_id=54b9e338f22db30e3c0155ad)

## Stacktrace

    lib/twitter_list_updater.rb:18 - block in add_missing_to_list
    lib/twitter_list_updater.rb:17 - each
    lib/twitter_list_updater.rb:17 - add_missing_to_list
    lib/twitter_list_updater.rb:11 - block in update
    lib/twitter_list_updater.rb:10 - update
    lib/tasks/twitter.rake:10 - block (2 levels) in &lt;top (required)&gt;

[View full stacktrace](https://bugsnag.com/hacken-dot-in/hacken-dot-in/errors/54b9e3380eb4aaa7d97168db?event_id=54b9e338f22db30e3c0155ad)

### PR DESCRIPTION
**Pull request by bitboxer**
The problem is non existing twitter handles. Now bugsnag will
tell us exactly which user is missing and we can fix the data
in our database.

### CODE REVIEW

File: lib/twitter_follower.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ def follow
         @client.follow(handle)
       rescue => e
         puts "Problems following #{handle}"
+        Bugsnag.notify "Problems following #{handle}"
```
Comment (moonglum) [inline lib/twitter_follower.rb]: Can we remove the `puts` Statement? Because now we notify instead?

File: lib/twitter_list_updater.rb [hunk 2]
```diff
@@ -17,8 +17,13 @@ def update
...
     handles_in_list = members(list).map(&:screen_name)
     (handles - handles_in_list).each do |screen_name|
-      request_wrapper do
-        @client.add_list_member(list, screen_name) unless screen_name == "hacken_in"
+      begin
+        request_wrapper do
+          @client.add_list_member(list, screen_name) unless screen_name == "hacken_in"
+        end
+      rescue => e
+        put "Problems adding #{screen_name} to the list"
```
Comment (moonglum) [inline lib/twitter_list_updater.rb]: "put"? Not "puts"? Also: See above.

**Comment by moonglum** [comment]
Apart from the two comments: Ready to go!

**Comment by moonglum** [comment]
And we already got our first reports, awesome!

**Comment by bitboxer** [comment]
Yes, the problem is that we have outdated twitter accounts in our system that don't exist anymore. We could automatically remove them, but that would mean if the twitter api is wonky, we loose all twitter data :cry:

**Comment by moonglum** [comment]
So for now we just remove them when an error occurs?

**Comment by bitboxer** [comment]
I think so. Really stupid, I know. If you have another solution, I would love to hear that.

**Comment by moonglum** [comment]
I don't think it's stupid. I think it's totally fine for now. This will not be a high traffic thing. And it allows the person that adjusts the record in the DB to see if there is a new twitter account for this specific event that should be followed instead.

**Pull request merged**